projects/storefrontstyles/scss/layout/page-templates/_cart-page.scss

```diff
@@ -15,13 +15,19 @@
 
   .CenterRightContentSlot {
     --cx-flex-basis: 30%;
+    flex-direction: column;
```

Comment:
It's considered a breaking change, if we can't find workaround around this, we would have to apply those changes to a new theme.

Reply:
Comment above.